cloud/aws/main.tf

```diff
@@ -83,7 +84,7 @@ resource "null_resource" "cockroach-initializer" {
     inline = [
       "bash download_binary.sh cockroach/cockroach ${var.cockroach_sha}",
       "mkdir -p logs",
-      "./cockroach init --logtostderr=true --stores=sdd=data",
+      "./cockroach init --logtostderr=true --stores=${element(split(",", "${var.stores}"), 0)}",
```

Comment:
Ok, this will actually try to bootstrap both stores which dies on start, so we need the `element(..., 0)`. Waiting for explanation of your `=` comment before merge (and maybe we should open an issue about the bootstrapping process).